src/olympia/pages/templates/pages/shield_study_4.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Enhance Websites</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_4/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Enhance Websites</h1>
+          <a href=https://addons.mozilla.org/firefox/downloads/latest/site-enhance-shield-study/?src=shield-study-4><button class="button primary large">Try Site Enhance</button></a>
```

Comment:
Why do you have the `button` inside the `a`? Can you not style the link instead?